index.bs

```diff
@@ -5720,6 +5736,26 @@ closed</dfn> steps given |window|, |type|, |accepted| and optional |user text|
    </dd>
 </dl>
 
+<div algorithm>
+To <dfn>get navigable's user prompt handler</dfn> given |type| and |navigable|:
+
+1. Let |user context| be |navigable|'s [=associated user context=].
+
+1. If [=unhandled prompt behavior overrides map=] contains |user context|:
+
+  1. Let |behavior map| be [=unhandled prompt behavior overrides map=][|user context|].
```

Comment:
Entries for this overrides map are not a map but a string or null at the moment.

Reply:
The `|behavior map|` at this point is an `unhandled prompt behavior` struct.

Reply:
I renamed it a bit to avoid confusion